include/global.h

```diff
@@ -167,8 +167,14 @@ struct UCoords32
     u32 y;
 };
 
+#include "constants/items.h"
+#define ITEM_FLAGS_COUNT ((ITEMS_COUNT / 8) + ((ITEMS_COUNT % 8) ? 1 : 0))
+
 struct SaveBlock3
 {
+#if OW_SHOW_ITEM_DESCRIPTIONS == OW_ITEM_DESCRIPTIONS_FIRST_TIME
+    /*0x000*/ u8 itemFlags[ITEM_FLAGS_COUNT];
```

Comment:
This may be just me, but I would personally inclined not to add offset comments here because they're fairly vestigial and context-dependent in the normal saveblocks too, and especially in saveblock3 where we're basically lumping all our new optional off-by-default stuff

Reply:
I agree. We already have a lot of wrong labels that we should remove so it makes sense not to have them at all.